diff --git a/mozjs/config/recurse.mk b/mozjs/config/recurse.mk
index 0d6102fac2..7b2a1746a7 100644
--- a/mozjs/config/recurse.mk
+++ b/mozjs/config/recurse.mk
@@ -217,10 +217,10 @@ $(addprefix build/unix/stdc++compat/,target host) build/clang-plugin/host: confi
 # prior to Make running. So we also set it as a dependency of pre-export, which
 # ensures it exists before recursing the rust targets and the export targets
 # that run cbindgen, tricking Make into keeping them early.
-$(rust_targets) gfx/webrender_bindings/export layout/style/export xpcom/base/export: $(DEPTH)/.cargo/config
-ifndef TEST_MOZBUILD
-pre-export:: $(DEPTH)/.cargo/config
-endif
+#$(rust_targets) gfx/webrender_bindings/export layout/style/export xpcom/base/export: $(DEPTH)/.cargo/config
+#ifndef TEST_MOZBUILD
+#pre-export:: $(DEPTH)/.cargo/config
+#endif
 
 # When building gtest as part of the build (LINK_GTEST_DURING_COMPILE),
 # force the build system to get to it first, so that it can be linked
