diff --git a/mozjs/build/moz.configure/bindgen.configure b/mozjs/build/moz.configure/bindgen.configure
index 335768ffe2..8536aae96d 100644
--- a/mozjs/build/moz.configure/bindgen.configure
+++ b/mozjs/build/moz.configure/bindgen.configure
@@ -238,6 +238,9 @@ set_config('MOZ_CLANG_PATH', bindgen_config_paths.clang_path)
          bindgen_config_paths.clang_flags)
 def basic_bindgen_cflags(target, is_unix, compiler_info, android_cflags,
                          clang_flags):
+    if not clang_flags:
+        return []
+
     args = [
         '-x', 'c++', '-fno-sized-deallocation', '-fno-aligned-new',
         '-DTRACING=1', '-DIMPL_LIBXUL', '-DMOZILLA_INTERNAL_API',
